### DISCUSSION TITLE
How to uncheck all checkbox after a trigger

### DISCUSSION THREAD

**Discussion by DanPresa** [KevinVandy/material-react-table#148] [Q&A]
I have to select some or maybe all checkbox and add them to an array and then submit them with a form but. I'm not able to uncheck all selected checkbox after submit.
I'm using accessorFn to show a checkbox in a column.
Does someone have an idea?

**Comment by KevinVandy** [comment]
So you are not using the built-in row selection features? https://www.material-react-table.com/docs/guides/row-selection

**Reply by KevinVandy** [reply]
If you were using the built in row-selection feature, you would just have to set the `rowSelection` state to an empty object `{}` to reset the state, or call `table.resetRowSelection()` from the table instance.

If you are using your own implementation, then just simply reset your state to whatever? Make sure you are forcing a re-render too

**Reply by DanPresa** [reply]
That sounds good. Let  me try it out.
Thanks a lot
